modules/core/src/main/scala/org/tessellation/Main.scala

```diff
@@ -43,25 +42,23 @@ object Main
         p2pClient = P2PClient.make[IO](sdkP2PClient, sdkResources.client, sdkServices.session)
         queues <- Queues.make[IO](sdkQueues).asResource
         storages <- Storages.make[IO](sdkStorages).asResource
-        services <- Services.make[IO](sdkServices, queues).asResource
+        services <- Services.make[IO](sdkServices, queues, storages, sdk.nodeId, keyPair, cfg).asResource
         programs = Programs.make[IO](sdkPrograms, storages, services)
         validators = Validators.make[IO](cfg.snapshot)
         healthChecks <- HealthChecks
           .make[IO](storages, services, p2pClient, cfg.healthCheck, sdk.nodeId)
           .asResource
 
-        _ <- services.stateChannelRunner.initializeKnownCells.asResource
+        streams = Streams.make(storages, validators, queues, cfg)
```

Comment:
shouldn't it be created inside a Daemon.make? It's a daemon itself as you said today

Reply:
Ok, I'm removing `Streams` altogether and moving stream creation to `Daemons`.